### ISSUE TITLE
[Bug] #8858, Model browser sometimes ignores mouse selections

### ISSUE THREAD

**Issue by JamesT-W** [ficool2/HammerPlusPlus-Issue-Tracker#178]
Has happened for a while now in Hammer++ across various versions.

Sometimes it's impossible to select models in the model browser with your mouse. Double clicking a model just closes the window and does not change the model selected.

A workaround is to use the arrow keys and pressing enter, but it isn't ideal

**Comment by MARK2580** [comment]
Yes, sometimes I noticed this behavior, I just go again, select the model and click OK, and not x2 click on the model.

**Comment by DrinklyDram** [comment]
you should press shift, ctrl (and maybe alt) one time to "fix" this error, because they are somehow getting sticky, i thought my keyboard was broken

**Comment by Qdude21** [comment]
Yeah this happens to me, too. Instead of simply clicking on the model, it acts like I'm holding alt or ctrl and does those functions accordingly, when all I want to do is click.
